### ISSUE TITLE
Issue creating new project with yarn 1.0.1

### ISSUE THREAD

**Issue by mnutt** [glimmerjs/glimmer-blueprint#63]
I'm having an issue creating a new glimmer app while using the newly released yarn 1.0.1 and node 8.4.0.

(If this would be better filed as a yarn issue please let me know and I'll move it there.)

```
> ember new traffic-viz -b @glimmer/blueprint                                                                                                                                
installing blueprint
  create .editorconfig
  create .watchmanconfig
  create README.md
  create config/environment.js
  create config/module-map.d.ts
  create config/resolver-configuration.d.ts
  create config/targets.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public/robots.txt
  create src/index.ts
  create src/main.ts
  create src/ui/components/traffic-viz/component-test.ts
  create src/ui/components/traffic-viz/component.ts
  create src/ui/components/traffic-viz/template.hbs
  create src/ui/index.html
  create src/ui/styles/app.scss
  create src/utils/test-helpers/test-helper.ts
  create testem.json
  create tsconfig.json
  create yarn.lock
Error creating new application. Removing generated directory `./traffic-viz`
Command failed: yarn install --ignore-optional --verbose --non-interactive
warning "ember-build-utilities@0.4.0" has incorrect peer dependency "@glimmer/compiler@> 0.26.0".
warning "@glimmer/inline-precompile@1.0.1" has incorrect peer dependency "@glimmer/compiler@> 0.23.0 || > 0.24.0-alpha.1 || > 0.25.0-alpha.1".
warning "babel-plugin-glimmer-inline-precompile@1.2.0" has incorrect peer dependency "@glimmer/compiler@> 0.23.0 || > 0.24.0-alpha.1 || > 0.25.0-alpha.1".
warning "@glimmer/test-helpers@0.30.0" has incorrect peer dependency "@glimmer/component@>= 0.4.0".
error /Users/michael/p/movableink/traffic-viz/node_modules/node-sass: Command failed.
Exit code: 1
Command: node scripts/install.js
Arguments: 
Directory: /Users/michael/p/movableink/traffic-viz/node_modules/node-sass
Output:
module.js:491
    throw err;
    ^

Error: Cannot find module 'process-nextick-args'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/michael/p/movableink/traffic-viz/node_modules/readable-stream/lib/_stream_readable.js:26:23)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
```

(the full debug output is ~20k lines and I didn't see anything relevant, but if it helps, it's here: https://gist.github.com/mnutt/5978d98b0dedb0cc7efe157c889e922a)

I've tried `yarn cache clean`, removing all global yarn modules, etc. When I roll back to yarn 0.27.5, the issue goes away.

It appears that we're installing both `node-sass` 4.5.3 and 3.13.1, and unfortunately it's hard to determine which one is causing the issue. I'm able to successfully `yarn global add node-sass@4.5.3` and `yarn global add node-sass@3.13.1` but it doesn't fix the issue.

**Comment by josemarluedke** [comment]
I can confirm this error. Having the same problem.

**Comment by josemarluedke** [comment]
@mnutt You can go around this by using npm to generate your app: `ember new traffic-viz -b @glimmer/blueprint --yarn=false`.

**Comment by t-sauer** [comment]
Seems like there was a problem with yarn that messed up nested dependencies when using `--skip-npm` (which Ember-CLI uses) which is already fixed: https://github.com/yarnpkg/yarn/pull/4448

**Comment by t-sauer** [comment]
Problem is fixed in yarn 1.1, so I guess this issue can be closed.

**Comment by mnutt** [comment]
Thanks all, it works now with yarn 1.1.
